<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单位换算器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="convertedData">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>29</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="originalData">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(160, 160, 160)</string>
       </property>
       <property name="text">
        <string>0 =</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">.QComboBox{
	border: 1px solid black;
}

.QLineEdit{
	border-radius: 10%;
	border: 1px solid black;
}

.QComboBox:hover{
	background-color: rgb(224, 238, 249);
}

.QLineEdit:hover{
	background-color: rgb(224, 238, 249);
}

</string>
     </property>
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
